spec/requests/authentication_spec.rb

```
@@ -71,11 +68,8 @@
           )
         end
         let(:credentials) { { user_name: user.name, password: 'wrongpassword' } }
-        let(:token) { JsonWebToken.encode({id: user.id}) }
-        let(:Authorization) { "Bearer #{token}" }
-        let(:headers) { { "Authorization" => "Bearer #{token}" } }
         run_test!
       end
     end
   end
-end
+end
```

RuboCop (1.86.1)

[convention] Final newline missing.

(Layout/TrailingEmptyLines)